UKHomeOffice/cerberus-service
# Cop 9849 - Display check-in time as "x days/hours before arrival"

## Description
This PR added the relative time between check-in time and arrival time (eta).

## To Test
- Pull & run against dev.
- Compare the `Booking and check-in` section of the UI (sections circled in red)

> Old
![check-in-section-old](https://user-images.githubusercontent.com/19333750/150161936-84788c8c-0f43-4a23-a436-7a1a75c6dd1b.png)

> Updated
![check-in-section-new](https://user-images.githubusercontent.com/19333750/150162184-37b8c560-ec17-4bc6-824a-90709efe52ac.png)




## Developer Checklist

\* Required
